cpp/daal/include/algorithms/tree_utils/tree_utils.h

```diff
@@ -59,6 +59,7 @@ struct DAAL_EXPORT SplitNodeDescriptor : public NodeDescriptor
 {
     size_t featureIndex; /*!< Feature used for splitting the node */
     double featureValue; /*!< Threshold value at the node */
+    double coverValue;   /*!< Cover (sum_hess) for the node */
```

Comment:
let's try to make our public headers more understandable
```suggestion
    double coverValue;   /*!< Cover, a sum of the Hessian values of the loss function evaluated at the points flowing through the node */
```